FWCore/ServiceRegistry/test/serviceregistry_t.cppunit.cpp

```diff
@@ -73,75 +73,64 @@ namespace {
 void testServiceRegistry::externalServiceTest() {
   edm::AssertHandler ah;
 
-  {
-    auto dummyPtr = std::make_unique<DummyService>();
-    dummyPtr->value_ = 2;
-    edm::ServiceToken token(edm::ServiceRegistry::createContaining(std::move(dummyPtr)));
-    {
-      edm::ServiceRegistry::Operate operate(token);
-      edm::Service<DummyService> dummy;
-      CPPUNIT_ASSERT(dummy);
-      CPPUNIT_ASSERT(dummy.isAvailable());
-      CPPUNIT_ASSERT(dummy->value_ == 2);
-    }
-    {
-      std::vector<edm::ParameterSet> pss;
-
-      edm::ParameterSet ps;
-      std::string typeName("DummyService");
-      ps.addParameter("@service_type", typeName);
-      int value = 2;
-      ps.addParameter("value", value);
-      pss.push_back(ps);
-
-      edm::ServiceToken token(edm::ServiceRegistry::createSet(pss));
-      edm::ServiceToken token2(
-          edm::ServiceRegistry::createContaining(std::move(dummyPtr), token, edm::serviceregistry::kOverlapIsError));
-
-      edm::ServiceRegistry::Operate operate(token2);
-      edm::Service<testserviceregistry::DummyService> dummy;
-      CPPUNIT_ASSERT(dummy);
-      CPPUNIT_ASSERT(dummy.isAvailable());
-      CPPUNIT_ASSERT(dummy->value() == 2);
-    }
-  }
+  {{auto dummyPtr = std::make_unique<DummyService>();
```

Comment:
This spacing looks weird. Did clang format do this?

Reply:
clang format did this spacing. I don't like it either. Just now, I tried
backing it out and redoing the indentation as I like it, but that
didn't help. At least in my working area, "scram b code-format"
put this weird formatting back in. I do not see any problem in my
code causing it. I would like to see every open curly brace on its
own line with increasing  indentation inside each one...